index.bs

```diff
@@ -1526,31 +1511,25 @@ The <dfn attribute for="XRInputSource">gripSpace</dfn> attribute is an {{XRSpace
+Similarly, the Valve Index controller is backwards compatible with the HTC Vive controller, but the Index controller has additional buttons and axes. As a result, the {{XRInputSource/profiles}} for the Valve Index controller could be: <code>["valve-index", "htc-vive", "touchpad-thumbstick-controller"]</code>. In this case the input layout described by the <code>"valve-index"</code> profile is a superset of the layout described by the <code>"htc-vive"</code> profile. Also, the <code>"valve-index"</code> profile indicates the precise appearance of the controller, while the <code>"htc-vive"</code> controller has a significantly different appearance. In this case the UA would have deemed that difference acceptable. And as in the first example, the last profile is a generic fallback.
 
 (Exact strings are examples only. This specification does not make any recommendation for profile strings.)
 </p>
 
-The <dfn attribute for="XRInputSource">gamepad</dfn> attribute is a {{Gamepad}} that describes the state of any buttons and axes on the [=XR input source=]. If the  [=XR input source=] does not have at least one of the following properties, the {{gamepad}} attribute MUST be <code>null</code>:
-
- - Multiple buttons
- - A button which reports analog values
- - An axis
-
-Note: {{XRInputSource}}s in an {{XRSession}}'s {{XRSession/inputSources}} array are "live". As such values within them, such as the {{gamepad}} button and axis state, are updated in-place. This means that it doesn't work to save a reference to an {{XRInputSource}}'s {{gamepad}} on one frame and compare it to the same {{XRInputSource}}'s {{gamepad}} from a subsequent frame to test for state changes, because they will be the same object. Therefore developers that wish to compare input state from frame to frame should copy the state in question.
+Note: {{XRInputSource}}s in an {{XRSession}}'s {{XRSession/inputSources}} array are "live". As such values within them, such as the {{gripSpace}}, are updated in-place. This means that it doesn't work to save a reference to an {{XRInputSource}}'s {{gripSpace}} on one frame and compare it to the same {{XRInputSource}}'s {{gripSpace}} from a subsequent frame to test for state changes, because they will be the same object. Therefore developers that wish to compare input state from frame to frame should copy the state in question.
```

Comment:
This is slightly less effective now, because there's not really an observable state on the `gripSpace` until you query it's pose, which isn't the same type of "live" behavior. I'm wondering if this whole paragraph should just move to the gamepads module?

Reply:
yeah I was going back and forth on that because the actual behavior isn't different in the gamepads module.  I tried to split the difference by keeping the original text in that repo and making a small modification here.  Perhaps if I just remove the example here?

Reply:
Yeah, that seems best. You've got an explanation of the "liveness" of the gamepad attrib in that module anyway.

Reply:
If this modification doesn't work for you, unresolve and I'll take another stab at it.